CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ### Changed
 
+BC-237 - Clean up global entity registration
```

Comment:
I think it contain some more changes like into factorys, changing default handling. The changelog is for other developers a good hint to see which changes are into it. Maybe you can increase the information.